beginner_source/former_torchies_tutorial.rst

```diff
@@ -25,9 +25,6 @@ In this tutorial, you will learn the following:
 .. galleryitem:: /beginner/former_torchies/tensor_tutorial.py
     :figure: /_static/img/tensor_illustration_flat.png
 
-.. galleryitem:: /beginner/former_torchies/autograd_tutorial.py
```

Comment:
@soumith Why is this removed? former_torchies/autograd_tutorial.py itself is edited as a part of the PR. Should I add this back?

Reply:
yes please, add this back. Not sure what happened here. cc: @SsnL

Reply:
added back now.

Reply:
thanks! Will build the latest docs.